sql/1.107/github_commit_comments_table.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+CREATE TABLE "github_commit_comment" (
+    "id" serial NOT NULL,
```

Comment:
I was following the pattern of other `github_*` table creation scripts, like this one.
https://github.com/duckduckgo/community-platform/blob/886b09013a03c99ebf3cb94e1426ceb731a6f550/sql/1.044/github_tables.sql